qemu/tests/virtio_fs_bad_socket_group_name.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import shutil
+import re
+
+from virttest import data_dir
+from virttest import error_context
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Negative test.virtiofsd shouldn't boot up with an unknown socket group name.
+    Steps:
+        1. Create shared directories on the host.
+        2. Run virtiofsd daemons with an unknown socket group name.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    result_pattern = params.get("result_pattern", "unable to find group")
+
+    # set fs daemon path
+    fs_source = params.get('fs_source_dir')
+    base_dir = params.get('fs_source_base_dir', data_dir.get_data_dir())
+
+    if not os.path.isabs(fs_source):
+        fs_source = os.path.join(base_dir, fs_source)
+    if os.path.exists(fs_source):
+        shutil.rmtree(fs_source, ignore_errors=True)
+    logging.info("Create filesystem source %s.", fs_source)
+    os.makedirs(fs_source)
+
+    sock_path = os.path.join(data_dir.get_tmp_dir(),
+                             '-'.join(('avocado-vt-vm1', 'viofs', 'virtiofsd.sock')))
+    # run daemon
+    process.system(params.get('cmd_create_fs_source'), shell=True)
+    cmd_run_virtiofsd = params.get('cmd_run_virtiofsd') % sock_path
+    cmd_run_virtiofsd += ' -o source=%s' % fs_source
+    cmd_run_virtiofsd += params.get('fs_binary_extra_options')
+
+    error_context.context('Running daemon command %s.' % cmd_run_virtiofsd, logging.info)
```

Comment:
Now I can’t find the message in debug.log, because no guest is started, so there is no session.log.
@YongxueHong Any good suggestions?
`log = logging.getLogger('avocado.test')`  what do you think?

Reply:
Hi @zhenyzha ,  you mean error_context.context() can only be used when guest is started?
Thanks.

Reply:
I just run it and can have the context log.
2022-01-04 01:24:55,424 error_context    L0079 INFO | Context: Running daemon command /usr/libexec/virtiofsd --socket-path=/tmp/avocado_aysrpq53/avocado-vt-vm1-viofs-virtiofsd.sock -o source=/root/avocado/data/avocado-vt/virtio_fs_test/ --socket-group=viofs_g.

Reply:
Sorry, let me explain:
Currently `force_create_fs_source = yes`

So the following path is called:

https://github.com/avocado-framework/avocado-vt/blob/master/virttest/env_process.py#L175
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qdevices.py#L1796

Reply:
> Now I can’t find the message in debug.log, because no guest is started, so there is no session.log. @YongxueHong Any good suggestions? `log = logging.getLogger('avocado.test')` what do you think?

@YongxueHong Do you have any idea? Do I need to handle this kind of issue recently?

Reply:
I think that @zhenyzha 's means that if you use the latest avocado version then you will lose this part info in the debug.log
So can you have a try it in the latest avocado version?
Thanks.